classes/php.php

```diff
@@ -41,6 +41,11 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (basename($phpPath) === 'hhvm')
```

Comment:
If you make this change here, and remove [this redundant test](https://github.com/jubianchi/atoum/blob/5c894af0/tests/units/classes/iterators/recursives/directory/factory.php#L17) (see `testGetIterator`), you have everything that's in #479 and #480.

I offered my PRs as an alternative to this one, in case small increments towards working HHVM builds are easier to deal with than this big PR.

If you want to go forward with this one, mine are obsolete. I just seemed to me that nothing was happening here anymore.